torch/_inductor/codegen/rocm/rocm_template_buffer.py

```diff
@@ -1,13 +1,14 @@
-# mypy: allow-untyped-defs
-from ...ir import TemplateBuffer
+from typing import Any, Callable, Sequence
+
+from ...ir import Buffer, Layout, TemplateBuffer
 
 
 class ROCmTemplateBuffer(TemplateBuffer):
     def __init__(
         self,
-        layout,
-        inputs,
-        make_kernel_render,
+        layout: Layout,
+        inputs: Sequence[Buffer],
+        make_kernel_render: Callable[..., Any],
```

Comment:
Nit: Better to future spec with ParamSpec, and with a TypeVar return for the Callable